docset/winserver2022-ps/addsdeployment/Test-ADDSDomainControllerInstallation.md

```diff
@@ -347,12 +416,14 @@ Accept wildcard characters: False
 ```
 
 ### -MoveInfrastructureOperationMasterRoleIfNecessary
-Indicates that the cmdlet transfers the infrastructure master role to the domain controller that you create in case the transfer is needed.
-You cannot use the *NoGlobalCatalog* parameter when specifying this parameter.
-Do not specify this parameter if you want the infrastructure master role to remain where it currently is.
+
+Indicates that the cmdlet transfers the infrastructure master role to the domain controller that you
+create in case the transfer is needed. You cannot use the `-NoGlobalCatalog` parameter when
+specifying this parameter. Do not specify this parameter if you want the infrastructure master role
+to remain where it currently is.
```

Comment:
```suggestion
Indicates that the cmdlet transfers the infrastructure master role to the domain controller that
you create in case the transfer is needed. You cannot use the **NoGlobalCatalog** parameter when
specifying this parameter. Do not specify this parameter if you want the infrastructure master role
to remain where it currently is.
```